.pre-commit-config.yaml

```diff
@@ -1,9 +1,9 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 24.10.0
+    rev: 24.3.0
     hooks:
       - id: black
-        args: [--safe]
+        args: ["--line-length", "120", "--diff", "."]
```

Comment:
`--diff` argument only outputs the changes that would be made, but pre-commit does not display that output and it changes no files, so this step does nothing and the flake8 fails